### PR TITLE
docs: restructure changelog, fix typos, change formulation

### PR DESCRIPTION
**Pull request by Tim-M-S**
## Why:

Closes: #ID

## Checklist before requesting a review

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: docs/changelog/changelog_en.md [hunk 1]
```diff
@@ -19,56 +19,63 @@
...
+
+- The app now reads metadata for products and categories, from the environment.
+- The app now reads title and alternate for product images, from the environment.
+- The app now reads metadata, such as description and keywords, from the environment.
+- The app now reads the minimum required Open Graph data, that is the title, image, and endpoint, from the environment.
+- Added canonical meta tags to product items.
+
+#### Page speed
+
+- PayPal: We now use card fields instead of hosting fields for credit card payments.
```
Comment (maxiroellplenty) [inline docs/changelog/changelog_en.md]: not sure if this is page speed relevant

File: docs/changelog/changelog_en.md [hunk 2]
```diff
@@ -19,56 +19,63 @@
...
+- Added cache-control for all static images to improve caching policy.
+- PayPal button is lazyloaded to improve performance.
```
Comment (maxiroellplenty) [inline docs/changelog/changelog_en.md]: need to check if pp blocking the main thread negates this point

File: docs/changelog/changelog_en.md (hunk 1 above)
Comment (Tim-M-S) [reply docs/changelog/changelog_en.md]: done

File: docs/changelog/changelog_en.md (hunk 2 above)
Comment (Tim-M-S) [reply docs/changelog/changelog_en.md]: done

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**